src/extension-runners/chromium.js

```diff
@@ -151,6 +153,18 @@ export class ChromiumExtensionRunner {
       chromeFlags.push(...this.params.args);
     }
 
+    const { chromiumPort } = this.params;
+    if (chromiumPort) {
```

Comment:
Are you intentionally excluding `--chromium-port=0` here? The validation helpers that you have added are not 100% correct when `0` is passed.

The expected behavior when `0` is passed is to select a random port (which chrome-launcher does indeed).